app/src/main/java/ca/klapstein/baudit/data/Patient.java

```diff
@@ -3,15 +3,22 @@
 /**
  * Class that represents a Patient.
  */
-public class Patient extends Account {
+public class Patient extends Account implements Comparable<Patient> {
     private static final String TAG = "Patient";
     private ProblemTreeSet problemTreeSet;
-
+    
     /**
      * A {@code Patient} can only have one {@code BodyPhoto}.
      */
     private BodyPhoto bodyPhoto;
-
+    
+    public Patient(Username username, Password password, ContactInfo contactInfo){
```

Comment:
It may be useful to have the `Account` constructor handle setting the `username`, `password`, and `contactInfo`.

Look into how you can call `super(<init args>)` in java to simplify constructors among subclasses.